hapi/src/utils/lifebankcode.js

```diff
@@ -45,7 +45,7 @@ const addLifebank = (
         data: {
           account,
           lifebank_name: name,
-          has_immunity_test: has_immunity_test || false,
+          hasImmunityTest: hasImmunityTest || false,
```

Comment:
@JustinCast the smart contract action expect "has_immunity_test" not "hasImmunityTest" https://jungle3.bloks.io/account/lifebankcode?loadContract=true&tab=Actions&account=lifebankcode&scope=lifebankcode&limit=100&table=lifebanks&action=addlifebank if you want to fix a linter warning related to snake case the proper way will be `has_immunity_test: hasImmunityTest || false,` so that the smart contract get the value with the propper key because `hasImmunityTest` will be ignored because it's no registered in the action